packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -6,9 +6,9 @@
  */
 
 import React, {useState, useEffect} from 'react'
-import {Helmet} from 'react-helmet'
 import PropTypes from 'prop-types'
 import {useHistory, useLocation} from 'react-router-dom'
+import {StorefrontPreview} from '@salesforce/pwa-kit-react-sdk/ssr/universal/components/storefront-preview'
```

Comment:
Aside: To emphasize Storefront Preview as a first-class feature, maybe we should create an alias import for the file: `@salesforce/pwa-kit-react-sdk/storefront-preview`.